near-sdk/src/environment/env.rs

```diff
@@ -930,8 +1700,11 @@ pub fn storage_write(key: &[u8], value: &[u8]) -> bool {
 }
 /// Reads the value stored under the given key.
 ///
-/// # Examples
+/// # Use cases
```

Comment:
should mention usage in 
https://github.com/near/near-sdk-rs/tree/master/near-contract-standards/src/upgrade/mod.rs

Reply:
usage of this call is only present in pending pr https://github.com/near-examples/update-migrate-rust/pull/2